src/hotspot/share/gc/shared/oopStorageSet.cpp

```diff
@@ -86,7 +86,7 @@ bool OopStorageSet::print_containing(const void* addr, outputStream* st) {
   if (addr != nullptr) {
     const void* aligned_addr = align_down(addr, alignof(oop));
     for (OopStorage* storage : Range<Id>()) {
-      if (storage->print_containing((oop*) aligned_addr, st)) {
+      if (storage != nullptr && storage->print_containing((oop*) aligned_addr, st)) {
```

Comment:
Add a comment?  Something like "Might get here while handling error before storage initialization."

Reply:
Sure thing, see new commit.

Reply:
Another maybe preferable option could be printing "uninitialized" or something.

Reply:
@tschatzl I don't think printing "uninitialized" or anything like that is really appropriate here.  What this code
is doing is printing something if-and-only-if the pointer is found to be in an oopstorage block.  There aren't
any of those if there's no oopstorage yet.